acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/signer/runner/CmdLineParamsConfigFileImpl.java

```diff
@@ -295,7 +308,70 @@ private String createEth2SlashingProtectionArgs() {
     return yamlConfig.toString();
   }
 
-  private String createCommaSeparatedList(final List<String> values) {
+  private String awsSecretsManagerOptions() {
+    final StringBuilder yamlConfig = new StringBuilder();
+
+    if (signerConfig.getAwsSecretsManagerParameters().isPresent()) {
+      final AwsSecretsManagerParameters awsSecretsManagerParameters =
+          signerConfig.getAwsSecretsManagerParameters().get();
+      yamlConfig.append(
+          String.format(
+              YAML_BOOLEAN_FMT,
+              "eth2." + AWS_SECRETS_ENABLED_OPTION.substring(2),
+              awsSecretsManagerParameters.isAwsSecretsManagerEnabled()));
+
+      yamlConfig.append(
+          String.format(
+              YAML_STRING_FMT,
+              "eth2." + AWS_SECRETS_AUTH_MODE_OPTION.substring(2),
+              awsSecretsManagerParameters.getAuthenticationMode().name()));
+
+      yamlConfig.append(
+          String.format(
+              YAML_STRING_FMT,
+              "eth2." + AWS_SECRETS_ACCESS_KEY_ID_OPTION.substring(2),
+              awsSecretsManagerParameters.getAccessKeyId()));
+
+      yamlConfig.append(
+          String.format(
+              YAML_STRING_FMT,
+              "eth2." + AWS_SECRETS_SECRET_ACCESS_KEY_OPTION.substring(2),
+              awsSecretsManagerParameters.getSecretAccessKey()));
+
+      yamlConfig.append(
+          String.format(
+              YAML_STRING_FMT,
+              "eth2." + AWS_SECRETS_REGION_OPTION.substring(2),
+              awsSecretsManagerParameters.getRegion()));
+
+      if (!awsSecretsManagerParameters.getPrefixesFilter().isEmpty()) {
```

Comment:
Are these if checks for empty list necessary? Won't the createCommaSeparatedList work on an empty list?

Reply:
I wanted to avoid specifying parameter with empty value i.e. `--aws-filter= --aws-filter=`